### PR TITLE
Allow for installation of LATEST_PACKAGE_ALIAS (*)

### LINKED ISSUE #190: Allow packages in bleeding to deploy latest (pre) release instead of master
@sondreso commented on [Thu Jan 28 2021](https://github.com/equinor/komodo-releases/issues/1121)

Using --pre with pip allows you to install latest release including pre releases (https://pip.pypa.io/en/stable/reference/pip_install/#install-pre). 

Think it would make sense to allow this option in bleeding in addition to master

### PR DESCRIPTION
**Pull request by jondequinor**
Fixes #190

### CODE REVIEW

File: komodo/package_version.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
+import re
+import subprocess
+import sys
+
+
+LATEST_PACKAGE_ALIAS = "*"
+_PYPI_LATEST_VERSION_RE = r".+from\ versions\:\ (.+)\)"
+# This command is deprecated. Hopefully it is not removed until a replacement
+# is made. For updates on this, see https://github.com/pypa/pip/issues/9139
+_PYPI_LATEST_VERSION_CMD = "python -m pip install --use-deprecated=legacy-resolver {}=="
```
Comment (sondreso) [inline komodo/package_version.py:10]: should this command also have `--pre`?

Comment (jondequinor) [reply komodo/package_version.py:10]: No

Comment (jondequinor) [reply komodo/package_version.py:10]: As seen here 
```
$ python -m pip install --use-deprecated=legacy-resolver equinor_libres==
… from versions: 3.3.1a9, 6.0.0a0, 7.1.0, 7.2.0b0, 7.2.0rc0, 7.2.0, 8.0.0b0, 8.0.0rc0, 8.0.0, 8.0.1, 8.1.0rc0, 8.1.0rc1, 8.1.0, 8.2.0b0, 9.0.0b0, 9.0.0b1, 9.0.0rc0
ERROR: No matching distribution found for equinor_libres==
$ python -m pip install --pre  --use-deprecated=legacy-resolver equinor_libres==
… from versions: 3.3.1a9, 6.0.0a0, 7.1.0, 7.2.0b0, 7.2.0rc0, 7.2.0, 8.0.0b0, 8.0.0rc0, 8.0.0, 8.0.1, 8.1.0rc0, 8.1.0rc1, 8.1.0, 8.2.0b0, 9.0.0b0, 9.0.0b1, 9.0.0rc0
```

**Review by sondreso — APPROVED** [review]
LGTM, nice initiative! 👏

**Pull request merged**